draft-ietf-mls-protocol.md

```diff
@@ -118,27 +118,27 @@ Instructions are on that page as well. Editorial changes can be
 managed in GitHub, but any substantive change should be discussed on
 the MLS mailing list.
 
-A group of agents who want to send each other encrypted messages needs
+Users who want to send each other encrypted messages needs
 a way to derive shared symmetric encryption keys. For two parties,
 this problem has been studied thoroughly, with the Double Ratchet
 emerging as a common solution {{doubleratchet}} {{signal}}.
-Channels implementing the Double Ratchet enjoy fine-grained forward secrecy as well as post-compromise
-security, but are nonetheless efficient enough for heavy use over
-low-bandwidth networks.
+Channels implementing the Double Ratchet enjoy fine-grained forward secrecy
+as well as post-compromise security, but are nonetheless efficient
+enough for heavy use over low-bandwidth networks.
 
 For a group of size greater than two, a common strategy is to
 unilaterally broadcast symmetric "sender" keys over existing shared
-symmetric channels, and then for each agent to send messages to the
+symmetric channels, and then for each Member to send messages to the
```

Comment:
Let's not capitalize terms, just be consistent.

Reply:
Likewise in a bunch of places below.

Reply:
I just removed capitalization everywhere.